frontend/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -85,7 +85,7 @@ class DateInput extends React.PureComponent<Props, State> {
 
     const style = { width }
     const label = element.get("label")
-    const minDate = new Date(element.get("min"))
+    const minDate = moment(element.get("min"), "YYYY/MM/DD").toDate()
```

Comment:
Not sure if its worth it but perhaps a comment that dates passed over are specifically formatted into this sequence.